src/quo2/components/drawers/bottom_actions/view.cljs

```diff
@@ -0,0 +1,39 @@
+(ns quo2.components.drawers.bottom-actions.view
+  (:require [quo2.components.buttons.button.view :as button]
+            [quo2.components.drawers.bottom-actions.style :as style]
+            [quo2.theme :as theme]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]))
+
+(defn- bottom-actions-internal
+  "Options:
+
+  :actions - keyword (default nil) - :1-action/:2-actions
+  :description - string (default nil) - Description to display below the title
+  :button-one-label - string (default nil) - Label for the first button
+  :button-two-label - string (default nil) - Label for the second button
+  :theme - :light/:dark
+  :blur? - bool (default false)  - Boolean to set the blur color variant
+  :scroll - bool (default false) - Whether the iOS Home Indicator should be rendered"
+  [{:keys [actions description button-one-label button-two-label theme blur?
+           scroll]}]
+  [rn/view
+   [rn/view {:style style/buttons-container}
+    (when (= actions :2-actions)
+      [button/button {:size            40
```

Comment:
there is a new convention in the codebase to use keywords in place of integers for sizes.
i.e here you should be using `:size/s-40` 

It seems no one added this to the docs so I will look to get it added this week 👍